ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.WebAPI/Controllers/TasksController.cs

```diff
@@ -47,5 +47,36 @@ public async Task<ActionResult<List<TaskDTO>>> GetTasksOnDateAsync([FromQuery, R
             var tasks = await _service.GetTasksOnDateAsync(date);
             return Ok(tasks);
         }
+
+        [HttpDelete("{id:guid}")]
+
+        public async Task<ActionResult> DeleteTaskAsync([FromRoute] Guid id)
+        {
+            await _service.DeleteTaskAsync(id);
+            return NoContent();
+        }
+
+        [HttpPut]
+        public async Task<ActionResult<TaskDTO>> UpdateTaskAsync([FromBody] TaskDTO task)
+        {
+            var taskDto = await _service.UpdateTaskAsync(task);
+            return Ok(taskDto);
+        }
+
+        [HttpPut("approve")]
+
+        public async Task<ActionResult> ApproveTasksAsync([FromQuery] DateTime date, [FromQuery] Guid projectId, [FromQuery] Guid employeeId)
+        {
+           await _service.ApproveTasksAsync(date, projectId, employeeId);
+           return NoContent();
+        }
+
+        [HttpPost("bulk")]
+
+        public async Task<ActionResult<List<CreateBulkTaskDTO>>> BulkCreateTasksDTOAsync([FromBody] CreateBulkTaskDTO tasks)
```

Comment:
rename tasks argument to bulkCreateTasksDto, tasks is more suitable for the list of tasks, but CreateBulkTaskDTO is not representing the list. From the name of var it should be clear it's type and purpose